.vscode/cspell.json

```diff
@@ -248,6 +248,13 @@
         "compilerArguments",
         "annotationProcessors"
       ]
+    },
+    {
+      "filename": "sdk/spring/CHANGELOG.md",
+      "words": [
+        "AADB",
```

Comment:
what is AADB?

Reply:
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/spring/CHANGELOG.md#breaking-changes-3

![image](https://user-images.githubusercontent.com/13167207/197493390-7e257589-303f-4f39-a664-866e8505fb67.png)